pkgs/tools/audio/whisper-ctranslate2/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, nix-update-script
+, python3
+}:
+let
+  pname = "whisper-ctranslate2";
+  version = "0.2.7";
+in
+python3.pkgs.buildPythonApplication {
+  inherit pname version;
```

```suggestion

python3.pkgs.buildPythonApplication rec {
  pname = "whisper-ctranslate2";
  version = "0.2.7";
```
I often see this pointed out, but I don't think it needs to be changed necessarily.

---

Thanks for taking the time to mention it.
Im actually against rec, some people mention it as an anti pattern, some people love it. What i personally dont like is that you cant tell at a glance what is being used where. Using let in is slightly more verbose, but personally i find it clearer.

---

Thanks for telling me why.
> What i personally dont like is that you cant tell at a glance what is being used where.

I agree with this.